discover/portal/articles/210-setting-up/00-intro.markdown

```diff
@@ -0,0 +1,90 @@
+# Setting Up [](id=setting-up)
+
+If @product@ is anything, it's configurable. As the core is shrinking
+due to its increased modularity, it's important that all the applications in
+Liferay are also configurable. 
+
+Breaking it down, three types of applications must be configurable:
+
+1.  The portal itself
+2.  Liferay's native applications 
+3.  Custom applications
+
+To this end, Liferay's engineers have made the platform and its own applications
+configurable, and created a mechanism for developers to make their
+[applications configurable](/develop/tutorials/-/knowledge_base/7-1/configurable-applications).
+
+## Where Configuration Happens [](id=where-configuration-happens)
+
+Liferay's configuration takes place in the following places: 
+
+Properties files: properties files  that set default behavior may be included in
```

Comment:
Maybe we should mention properties files second since they have become over time less relevant and people should not need to rely to them as often as the configuration in the UI.